src/quo2/components/wallet/token_input/style.cljs

```diff
@@ -0,0 +1,44 @@
+(ns quo2.components.wallet.token-input.style
+  (:require
+    [quo2.foundations.colors :as colors]
+    [react-native.platform :as platform]))
+
+
+(defn main-container
+  [width]
+  {:padding-vertical 8
+   :width            width})
+(def amount-container
+  {:padding-horizontal 20
+   :padding-bottom     4
+   :height             36
+   :flex-direction     :row
+   :justify-content    :space-between})
+
+(def token
+  {:width  32
+   :height 32})
+
+(def text-input
+  {:font-size    27
+   :font-weight  "600"
+   :line-height  32
```

Comment:
Couldn't we use heading-1 style? https://github.com/status-im/status-mobile/blob/0a8993bbf19fc368e4f3f04ed5daf73062712203/src/quo2/foundations/typography.cljs#L19